language/testing-infra/transactional-test-runner/src/tasks.rs

```diff
@@ -218,6 +218,11 @@ pub enum SyntaxChoice {
     IR,
 }
 
+/// Translates the given Move IR module into bytecode, then prints a textual representation of that
+/// bytecode.
+#[derive(Debug, StructOpt)]
+pub struct PrintBytecodeCommand {}
```

Comment:
I'd probably name this "DisassembleCommand", with "//# disassemble" as the directive, just to be consistent with the name of the tool used. However this is mostly just my personal preference so it's up to you.

Reply:
I started with that command name as well! But I then had second-thoughts... disassembling implies that the command takes something "assembled" as input and disassembles that input. However, in fact we're taking in a `.mvir` Move IR program (and soon, as you mention above, a `.move` Move program), "compiling" or "assembling" it into bytecode, and then disassembling the bytecode and printing it out. So, for a time I used the command name "CompileAndDisassemble."

But after some more thought, I figured the simplest-sounding name I could think of would probably be best. I think people need some background knowledge to understand what "disassemble" means (that is, you need to know what an assembler or assembly language is, and how those things are analogous to Move bytecode), but "print bytecode" seems very straightforward to me (you still need to know that Move is compiled to bytecode, but you don't need to know about assembly & disassemblers).

I'm definitely open to changing this, though. Maybe "PrintAsBytecode?" Naming is hard... 😅